### ISSUE TITLE
Add missing colours to the palette

### ISSUE THREAD

**Issue by wheresrhys** [Financial-Times/next-sass-setup#46]
- [ ] #b7a797 Or some equivalent. It's used for timestamps

**Comment by adgad** [comment]
Anything from the original o-colors palette (e.g #fff1e0, black/white etc) that we will be using in next should be added in the style guide also.

**Comment by keirog** [comment]
I spoke to Simon about this and it was a deliberate thing to omit o-colors' colors from the style guide. Will check with him about the rationale.

**Comment by simonwilliamsFT** [comment]
yes the o colours has too many muted beiges we intentionally want to steer away from these , we will be posting use cases for type and colours , topics , time stamps etc on style-guide  .  For now please use 505050 text sans 5,  capped
